File: java/src/main/java/id/my/niznet/tugas1/Person.java
package id.my.niznet.tugas1;

public class Person implements Human {
    public String nama, alamat;

    @Override
    public void jalan() {}

    @Override
    public void makan() {}

    @Override
    public void tidur() {}

    public String getNama() {
        return nama;
    }

    public void setNama(String nama) {
        this.nama = nama;
    }

    public String getAlamat() {
        return alamat;
    }

    public void setAlamat(String alamat) {
        this.alamat = alamat;
    }
}

File: kotlin/src/main/kotlin/id/my/niznet/tugas1/Person.kt
package id.my.niznet.tugas1

open class Person : Human {
    var nama: String? = null
        get() {
            return camelCase(field ?: "")
        }
        set(value) {
            field = camelCase(value ?: "")
        }
    var alamat: String? = null
    override fun jalan() {}
    override fun makan() {}
    override fun tidur() {}

    private fun camelCase(string: String, delimiter: String = " "): String {
        return string.split(delimiter).joinToString("") {
            it.lowercase().replaceFirstChar { char -> char.uppercase() }
        }
    }
}